wafer_space/projects/admin.py

```
@@ -223,5 +224,39 @@ class ManufacturabilityCheckAdmin(StaffReadOnlyAdminMixin, admin.ModelAdmin):
+@admin.register(PrecheckImageRevision)
+class PrecheckImageRevisionAdmin(admin.ModelAdmin):
+    """Admin for PrecheckImageRevision."""
+
+    list_display = [
+        "short_digest",
+        "precheck_version",
+        "git_commit_sha_short",
+        "first_seen_at",
+        "checks_count",
+        "metadata_fetched_at",
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)